core/src/test/java/com/opensymphony/xwork2/DefaultUnknownHandlerManagerTest.java

```diff
@@ -0,0 +1,69 @@
+package com.opensymphony.xwork2;
```

Comment:
Please copy and paste the LICENCE text from a neighbor test .java file to beginning of this file to pass Jenkins build :)

Reply:
thanks, this is clear and fixed.